CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+Welcome to nushell!
+
+*Note: for a more complete guide see https://github.com/nushell/contributor-book*
+
+
+For speedy contributions open it in GitPod nu will be preinstalled with the latest build in a vscode like editor all from your browser
+
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/nushell/nushell)
+
+To get live support from the community see our [Discord](https://discordapp.com/invite/NtAbbGn), [Twitter](https://twitter.com/nu_shell) or File an issue or request more features here on [Github](https://github.com/nushell/nushell/issues/new/choose)!
```

Comment:
I know the ordering of these things are a bit unorganized does anyone have an idea of how to structure it?